gsa/src/web/graphql/tags.js

```diff
@@ -145,3 +145,31 @@ export const useRemoveTag = () => {
   );
   return [removeTag, data];
 };
+
+export const BULK_TAG = gql`
+  mutation bulkTag($input: BulkTagInput!) {
+    bulkTag(input: $input) {
+      ok
+    }
+  }
+`;
+
+export const useBulkTag = () => {
+  const [queryBulkTag, data] = useMutation(BULK_TAG);
+  const bulkTag = useCallback(
+    // eslint-disable-next-line no-shadow
+    (tag_id, resourceType, resourceIds, resourceFilter) =>
```

Comment:
Personally I would put optional arguments into an object like
```js
(tag_id, resourceType, {resourceIds, resourceFilter})
```
in this case maybe it looks even nicer to group all resource arguments into the object like
```js
(tag_id, {resourceType, resourceIds, resourceFilter})